packages/api/amplify_api/lib/src/graphql/graphql_send_request.dart

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import 'dart:convert';
+
+import 'package:amplify_core/amplify_core.dart';
+import 'package:http/http.dart' as http;
+import 'package:meta/meta.dart';
+
+import '../util.dart';
+import 'graphql_response_decoder.dart';
+
+@internal
+Future<GraphQLResponse<T>> graphQLSendRequest<T>({
```

Comment:
nit: could you please add a little doc comment for this function saying what it does. I think this file will have no lint warnings after that. maybe something like:

```
/// Converts the [GraphQLRequest] to an HTTP POST request and sends with ///[client]. 
```

Reply:
+1. I would also rename it to `sendGraphQLRequest` per Dart [naming conventions](https://dart.dev/guides/language/effective-dart/design#prefer-an-imperative-verb-phrase-for-a-function-or-method-whose-main-purpose-is-a-side-effect).